### PR TITLE
[sjcl] create sjcl package

### PR DESCRIPTION
**Pull request by upgradingdave**
New package:

**Extern:** Generated.

This is a dependency for the AWS Cognito Identity SDK (I'll submit another pull request for that)

### CODE REVIEW

File: sjcl/build.boot [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+(def +lib-version+ "1.0.6")
+(def +version+ (str +lib-version+ "-1"))
+
+(task-options!
+  push {:ensure-clean false}
+  pom  {:project     'cljsjs/sjcl
+        :version     +version+
+        :description "Stanford Javascript Crypto Library"
+        :url         "https://github.com/bitwiseshiftleft/sjcl/blob/master/sjcl.js"
```
Comment (miikka) [inline sjcl/build.boot]: `:url` is usually a link to the project home page.

File: sjcl/build.boot [hunk 2]
```diff
@@ -0,0 +1,40 @@
+(set-env!
+  :resource-paths #{"resources"}
+  :dependencies '[[cljsjs/boot-cljsjs "0.5.2" :scope "test"]])
+
+(require '[boot.task-helpers]
+         '[cljsjs.boot-cljsjs.packaging :refer :all])
+
+
+(def +lib-version+ "1.0.6")
+(def +version+ (str +lib-version+ "-1"))
```
Comment (miikka) [inline sjcl/build.boot]: Customarily the CLJSJS package versions start at `"-0"`.

File: sjcl/build.boot [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+(task-options!
+  push {:ensure-clean false}
```
Comment (miikka) [inline sjcl/build.boot]: I don't think this is needed.

File: sjcl/build.boot [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  pom  {:project     'cljsjs/sjcl
+        :version     +version+
+        :description "Stanford Javascript Crypto Library"
+        :url         "https://github.com/bitwiseshiftleft/sjcl/blob/master/sjcl.js"
+        :license     {"BSD" "https://opensource.org/licenses/BSD-3-Clause"}
+        :scm         {:url "https://github.com/cljsjs/packages"}})
+
+(require '[boot.core :as c]
```
Comment (miikka) [inline sjcl/build.boot]: You're requiring a bunch of stuff here, but as far as I can tell, none of them are used anywhere.

File: sjcl/build.boot [hunk 5]
```diff
@@ -0,0 +1,40 @@
+(set-env!
+  :resource-paths #{"resources"}
+  :dependencies '[[cljsjs/boot-cljsjs "0.5.2" :scope "test"]])
+
+(require '[boot.task-helpers]
```
Comment (miikka) [inline sjcl/build.boot]: I don't think requiring `boot.task-helpers` here is needed. It's not explicitly used anywhere and Boot loads that namespace anyway.

**Comment by rarous** [comment]
There is misssing `deps.cljs` file

**Comment by upgradingdave** [comment]
Thanks for the feedback. Hope to have time to clean this up before the end of the week. I didn't see https://github.com/Sepia-Officinalis/sjcl-cljs, I'll change to use that as well.

**Comment by Deraen** [comment]
Closing due to inactivity. Please reopen if you want to finish this PR.

**Pull request closed without merging**